packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -7,7 +7,7 @@ import { Constants, Message } from "../constants";
 import { EndpointInvalidError, NoConfigError, VersionFileNotExist, ZipDownloadError } from "../errors";
 import { ResultFactory } from "../result";
 import { TelemetryUtils } from "./telemetry";
-import { BlobServiceClient } from "@azure/storage-blob";
+import downloadRelease from "download-github-release";
```

Comment:
Seems not an official package?
@adashen Can you use this kind of personal package?

Reply:
Better not to use this package. I just checked on npm, seems updated 4 years ago. Some vulnerable issue may be caused by introducing such kind of dependency. And it is not dev dependency.

Reply:
ok